config/params.php

```diff
@@ -19,7 +20,9 @@
     ],
     'yiisoft/yii-queue' => [
         'handlers' => [],
-        'channel-definitions' => [],
+        'channel-definitions' => [
+            QueueFactoryInterface::DEFAULT_CHANNEL_NAME => AdapterInterface::class,
```

Comment:
Should we fix readme [here](https://github.com/yiisoft/yii-queue#ready-for-yiisoftconfig)? Now `AdapterInterface` is optional.

Reply:
It's already fixed in `master`